evm_loader/lib/src/config.rs

```diff
@@ -1,18 +1,13 @@
 use std::{env, str::FromStr};
 
-use crate::rpc::CloneRpcClient;
-use crate::{types::ChDbConfig, NeonError};
+use crate::types::ChDbConfig;
 use serde::{Deserialize, Serialize};
-use solana_clap_utils::{
-    input_validators::normalize_to_url_if_moniker, keypair::keypair_from_path,
-};
-use solana_cli_config::Config as SolanaConfig;
-use solana_client::nonblocking::rpc_client::RpcClient;
 use solana_sdk::{commitment_config::CommitmentConfig, pubkey::Pubkey, signature::Keypair};
 
 #[derive(Debug)]
 pub struct Config {
     pub evm_loader: Pubkey,
+    pub operator: Pubkey,
     pub fee_payer: Option<Keypair>,
     pub commitment: CommitmentConfig,
     pub solana_cli_config: solana_cli_config::Config,
```

Comment:
`pub solana_cli_config: solana_cli_config::Config` can be removed, it's never used.

Reply:
Keep it here for now. Need to investigate whether we need to have the ability to read config from file or not

Reply:
Keep it here for now. We need a separate task to cleanup configuration options for CLI and API.